snuba/utils/streams/kafka.py

```diff
@@ -51,6 +51,10 @@ def __init__(self, state: KafkaConsumerState):
         self.__state = state
 
 
+def as_kafka_configuration_boolean(value: Union[str, int, bool]) -> bool:
+    return value in {"true", "t", "1", 1, True}
```

Comment:
can we add `# LOL` at the end of this line ? Reminds me of php:
```
echo  null == "";
true

echo  "1" != true;
false
```

Not making this up.

Reply:
I'd rather not, but this should probably contain a link to the reference implementation in the librdkafka source where these values were copied from.